js/widgets/help.js

```diff
@@ -101,27 +101,32 @@ class HelpWidget {
             leftArrow.classList.add("hover");
 
             let cell = docById("left-arrow");
-
+            if(page===0){
+                leftArrow.classList.add('disabled');
+            }
             cell.onclick = () => {
-                page = page - 1;
-                if (page < 0) {
-                    page = HELPCONTENT.length - 1;
-                }
-
-                this._showPage(page);
+                    if(page>0){
```

Comment:
could you please put spaces around operators to be consistent with the rest of the code? And space after if?